commit 2ebbf21fcd13d810bb7a2e6adde011f19033e9b7
Author: Cody Russell <cody@jhu.edu>
Date:   Sat Jul 8 15:17:28 2017 -0500

    [Mac] Use TISCopyCurrentASCIICapableKeyboardInputSource()
    
    Instead of TISCopyCurrentKeyboardLayoutInputSource(). This seems
    to fix issues with some Cyrillic keyboard layouts not being able to
    type shortcuts like Cmd-A, Cmd-S, etc.

diff --git a/gdk/quartz/gdkkeys-quartz.c b/gdk/quartz/gdkkeys-quartz.c
index 523aaf7e50..7d3e03c2b7 100644
--- a/gdk/quartz/gdkkeys-quartz.c
+++ b/gdk/quartz/gdkkeys-quartz.c
@@ -272,7 +272,7 @@ update_keymap (void)
    * 64-bit.
    */
 #ifdef __LP64__
-  TISInputSourceRef new_layout = TISCopyCurrentKeyboardLayoutInputSource ();
+  TISInputSourceRef new_layout = TISCopyCurrentASCIICapableKeyboardInputSource();
   CFDataRef layout_data_ref;
 
 #else
